### PR TITLE
Revert "Revert https://github.com/ASFHyP3/hyp3-docs/pull/346"

### PR DESCRIPTION
**Pull request by jtherrmann**
Reverts ASFHyP3/hyp3-docs#350

TODO:

- [x] Wait until the MintPy changes are released.
- [x] Update the HyP3 API URL to point at production hyp3.
- [x] Fix Changelog
- [x] Address items listed in https://asfdaac.atlassian.net/browse/TOOL-2176

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -104,7 +112,7 @@ and uses [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ### Changed
 * Replaced the Subscriptions documentation with a link to the new tutorials.
-* Upgraded the Conda environment to Python 3.9.
+* Upgraded the conda environment to Python 3.9.
```
Comment (jtherrmann) [inline CHANGELOG.md:115]:
```suggestion
* Upgraded the Conda environment to Python 3.9.
```

File: docs/tutorials/hyp3_isce2_burst_stack_for_ts_analysis.ipynb [hunk 2]
```diff
@@ -194,7 +194,7 @@
     "\n",
     "import hyp3_sdk as sdk\n",
     "\n",
-    "hyp3 = sdk.HyP3('https://hyp3-test-api.asf.alaska.edu', prompt=True)"
+    "hyp3 = sdk.HyP3('https://hyp3-api.asf.alaska.edu', prompt=True)"
```
Comment (jtherrmann) [inline docs/tutorials/hyp3_isce2_burst_stack_for_ts_analysis.ipynb:197]:
```suggestion
    "hyp3 = sdk.HyP3(prompt=True)"
```

File: mkdocs.yml [hunk 3]
```diff
@@ -82,6 +82,7 @@ nav:
     - Search another user's jobs: https://nbviewer.jupyter.org/github/ASFHyP3/hyp3-sdk/blob/main/docs/search_other_user_jobs.ipynb" target="_blank
     - Process new granules for search parameters: tutorials/process-new-granules-for-search-parameters.md
     - InSAR time series with MintPy: https://nbviewer.jupyter.org/github/ASFHyP3/hyp3-docs/blob/main/docs/tutorials/hyp3_insar_stack_for_ts_analysis.ipynb" target="_blank
+    - InSAR burst time series with MintPy: https://nbviewer.org/github/ASFHyP3/hyp3-docs/blob/main/docs/tutorials/hyp3_isce2_burst_stack_for_ts_analysis.ipynb" target="_blank
```
Comment (jtherrmann) [inline mkdocs.yml:85]: We'll also want to add the link to https://hyp3-docs.asf.alaska.edu/tutorials/

File: docs/tutorials.md [hunk 4]
```diff
@@ -14,7 +14,9 @@ We provide step-by-step tutorials for using HyP3 programmatically via Jupyter No
 * [Time series analysis with HyP3 and MintPy](https://nbviewer.org/github/ASFHyP3/hyp3-docs/blob/main/docs/tutorials/hyp3_insar_stack_for_ts_analysis.ipynb "Time series analysis with HyP3 and MintPy Tutorial" ){target=_blank}
   -- This notebook walks through performing a time-series analysis of the 2019 
   Ridgecrest, CA earthquake with HyP3 On Demand InSAR products and MintPy.
-
+* [Time series analysis with HyP3 ISCE2 burst InSAR products and MintPy](https://nbviewer.org/github/ASFHyP3/hyp3-docs/blob/main/docs/tutorials/hyp3_isce2_burst_stack_for_ts_analysis.ipynb "Time series analysis with HyP3 ISCE2 burst InSAR products and MintPy Tutorial" ){target=_blank}
+  -- This notebook walks through performing a time-series analysis of the 2019 
+  Ridgecrest, CA earthquake with HyP3 On Demand Burst InSAR products and MintPy.
```
Comment (jtherrmann) [inline docs/tutorials.md:20]:
```suggestion


```

**Review by jtherrmann — COMMENTED** [review]
LGTM, but I'd appreciate if @AndrewPlayer3 or @forrestfwilliams could give the tutorial a final read-through for correctness, as the original effort may have been a bit rushed.

**Comment by jtherrmann** [comment]
Trufflehog is complaining about a storymap link.

**Review by forrestfwilliams — APPROVED** [review]

**Pull request merged**